PIL/Image.py

```diff
@@ -667,6 +715,7 @@ def __setstate__(self, state):
         self.frombytes(data)
 
     def tobytes(self, encoder_name="raw", *args):
```

Comment:
Nothing to do with this particular PR, but from the typing, it seems that the function `_getencoder` is rather...strange. Like why do we need to _tweak_ arguments ?